src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -777,7 +777,7 @@ private void CheckCygwin(List<LaunchCommand> commands, LocalLaunchOptions localL
 
         private void AddExecutablePathCommand(IList<LaunchCommand> commands)
         {
-            string exe = EscapePath(_launchOptions.ExePath);
+            string exe = this.EscapePath(_launchOptions.ExePath);
```

Comment:
Aren't Android LaunchOptions not LocalLaunchOptions? I thought they were AndroidLaunchOptions